src/engraving/libmscore/shadownote.cpp

```diff
@@ -323,6 +324,7 @@ void ShadowNote::layout()
     // Layout ledger lines if needed
     if (!m_isRest && m_lineIndex < 100 && m_lineIndex > -100) {
         double extraLen = score()->styleMM(Sid::ledgerLineLength) * mag();
+        double step = _spatium * staffType()->lineDistance().val();
```

Comment:
Shouldn't this be `0.5 * _spatium`, to keep it similar to the draw method?

(Doesn't matter much anyway though, since the bbox calculated in this method is never really used, as far as I know.)

Reply:
You are absolutely right. Thanks.